Js/ligth_dark.js

```diff
@@ -0,0 +1,10 @@
+const modo_light_dark = document.querySelector('.icon-moon');
+
+modo_light_dark.addEventListener('click', () => {
+    modo_light_dark.classList.toggle('dark')
+    body.classList.toggle('dark')
+    header.classList.toggle('dark')
+});
```

Comment:
Acho que vc só precisa de uma classe no body pq vc pode usar a especificidade do CSS pra sobrescrever os estilos.

```suggestion
modo_light_dark.addEventListener('click', () => {
    body.classList.toggle('dark')
});
```

Reply:
Feito